controller_interface/include/semantic_components/force_torque_sensor.hpp

```diff
@@ -23,28 +23,29 @@
 #include "hardware_interface/loaned_state_interface.hpp"
 #include "semantic_components/semantic_component_interface.hpp"
 
+namespace
+{
+constexpr std::size_t FORCES_SIZE = 3;
+}
+
 namespace semantic_components
 {
```

Comment:
```suggestion
 namespace semantic_components
 {
 constexpr std::size_t FORCES_SIZE = 3;
```

Reply:
This one was intentional. When we use an anonymous namespace, the variables within this namespace are limited to this specific translation unit. Maybe it's unnecessary in this case.

Reply:
Yes! This might not be needed here.